### PR TITLE
fix(federation): use 'url' to fetch a space

### PR DESCRIPTION
**Pull request by mwegman**

### CODE REVIEW

**Comment by mwegman** [comment]
SPARK-114799 #done

File: packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js [hunk 1]
```diff
@@ -171,27 +171,25 @@ export function updateSpaceWithActivity(activity, isSelf, isReadable = false) {
  *
  * @export
  * @param {Object} sparkInstance
- * @param {String} spaceId
+ * @param {String} space
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js]: `space` is expected to be an object, correct?

File: packages/node_modules/@ciscospark/widget-space/src/actions.js [hunk 2]
```diff
@@ -154,7 +154,7 @@ export function getSpaceDetails({
 
     return sparkInstance.request({
       service: 'hydra',
-      resource: `/rooms/${spaceIdUUID}`
+      resource: `/rooms/${destinationId}`
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-space/src/actions.js]: I thought you said this doesn't work with a request to `/rooms/<hydra encoded url>`?

Comment (mwegman) [reply packages/node_modules/@ciscospark/widget-space/src/actions.js]: It doesn't work if it is a non 'US' cluster.

**Pull request merged**